app/controllers/photos_controller.rb

```diff
@@ -24,7 +25,7 @@ def edit
   def create
     @photo = Photo.new(photo_params)
     @photo.owner = current_user
-
+    
```

Comment:
For future reference, check the diff to prevent this: extra spaces were added and it shows as a diff. It's ok for now